### ISSUE TITLE
Enhancement: Customizable icon color for current folder bar

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/QuickFolders#317]
When creating a custom background for the current toolbar, the default color (currentColor) for the toolbars may not be ideal, so it would be much better to have a widget to control the font color separately. This should only be active when the "custom" background is being selected.

Example - a dark rainbow gradient in the default theme (black icon color) makes the icons hard to read:

![image](https://user-images.githubusercontent.com/6314674/193910932-2d99f0bd-9da7-4408-b18b-07cfbd1bffa7.png)

CSS value for this current folder toolbar example background is:
```css
linear-gradient(to right, rgba(221,107,0,1) 0%,rgba(201,164,0,1) 14%,rgba(0,128,0,1) 33%,rgba(0,128,0,1) 33%,rgba(0,10,155,1) 50%,rgba(75,0,130,1) 66%,rgba(160,25,151,1) 84%,rgba(181,0,0,1) 100%)
```

**Comment by RealRaven2000** [comment]
Trial version below. Select "custom" from the dropdown to apply the new icon color - I changed the fill opacity from 70% to 90% for a better result.

![image](https://user-images.githubusercontent.com/6314674/193932789-bde7a6f7-a6f6-4b55-9cde-2451a62128ee.png)

click on the icon color checkbox to activate the custom color.

![image](https://user-images.githubusercontent.com/6314674/193933058-b18dabe6-05b0-473d-9e36-d3b69054151b.png)


[QuickFolders-mx-5.13.2pre28.zip](https://github.com/RealRaven2000/QuickFolders/files/9710999/QuickFolders-mx-5.13.2pre28.zip)

**Comment by RealRaven2000** [comment]
I wrote a patch for also affecting the font color on the current folder tab in the center.

[QuickFolders-mx-5.13.2pre29.zip](https://github.com/RealRaven2000/QuickFolders/files/9714802/QuickFolders-mx-5.13.2pre29.zip)

Test it with the background value:
```css
linear-gradient(to right, rgba(184,0,0,1) 0%,rgba(184,110,0,1) 10%,rgba(151,160,24,1) 20%,rgba(39,177,35,1) 30%,rgba(33,171,169,1) 40%,rgba(24,154,174,1) 50%,rgba(14,91,178,1) 60%,rgba(67,10,180,1) 70%,rgba(135,5,183,1) 80%,rgba(183,3,159,1) 90%,rgba(184,0,0,1) 100%)
```

To install, download zip file and drag into Thunderbird Add-ons manager.

**Comment by RealRaven2000** [comment]
Implemented in 5.14 - Published 14/11/2022
